src/provider-strings.js

```diff
@@ -5,6 +5,9 @@
 // be detected.
 
 const PROVIDERS_STRINGS = {
+  frame: {
```

Comment:
Note that there's not a really good way to check for if we're connected to frame yet (_could_ try checking against `localhost:1234`) so this won't be used.